content/docs/beyond-the-basics/public-attestation/index.md

```diff
@@ -1,14 +1,165 @@
+
+{{< tabs name="create_asset_public" >}}
+{{{< tab name="UI" >}}
+Select `Add Asset` from the sidebar and fill in the desired details.
+{{< img src="CreateAsset.png" alt="Rectangle" caption="<em>Asset Details</em>" class="border-0" >}}
+Check the box next to `Make Asset Public`.
+{{< img src="PublicCheck.png" alt="Rectangle" caption="<em>Check Asset as Public</em>" class="border-0" >}}
+{{< /tab >}}
+{{< tab name="JSON" >}}
+Create a JSON file with your desired Asset details. Set keyword `public` to true.
+
+```json
+{
+    "behaviours": ["RecordEvidence", "Attachments"],
+    "attributes": {
+        "arc_display_name": "Publicly Attested Asset",
+        "arc_display_type": "Example",
+        "arc_description": "This example asset is publicly attested, so anyone with the link can access its details without signing in to RKVST."
+    },
+    "public": true
+}
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+2. Publish your Public Asset.
+
+{{< warning >}}
+**WARNING:** Once an Asset is made public, it cannot be made private. Any information published will be publicly accessible forever.
+{{< /warning >}}
+
+{{< tabs name="set_public_public" >}}
+{{{< tab name="UI" >}}
+Click `Create Asset` to complete your Public Asset creation. 
+{{< img src="PublicCheck.png" alt="Rectangle" caption="<em>Publish Your Asset</em>" class="border-0" >}}
+{{< /tab >}}
+{{< tab name="JSON" >}}
+Use the curl command to run your JSON file. See instructions for [creating your `BEARER_TOKEN_FILE`](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/) here. 
+
+```bash 
+curl -v -X POST \
+    -H "@$BEARER_TOKEN_FILE" \
+    -H "Content-type: application/json" \
+    -d "@/path/to/jsonfile" \
+    https://app.rkvst.io/archivist/v2/assets
+```
+
+{{< /tab >}}
+{{< /tabs >}}
+
+3. Retrieve public link to share your Public Asset with others. 
+
+{{< note >}}
+**NOTE:** A Public Asset may only be updated by the tenancy that created it. Anyone viewing the Asset using the public link will have read-only access.
+{{< /note >}}
+
+{{< tabs name="get_link_public" >}}
+{{{< tab name="UI" >}}
+Click on the copy icon next to the green `PUBLIC` badge. This will copy the Asset's public URL to your clipboard. 
+{{< img src="PublicAsset.png" alt="Rectangle" caption="<em>Copy the Public Link</em>" class="border-0" >}}
+{{< /tab >}}
+{{< tab name="JSON" >}}
+A Public Asset's URL can be retrived via the [Assets API](https://docs.rkvst.com/docs/api-reference/assets-api/). Use the Asset ID returned in the previous step.
+
+```bash
+curl -g -v -X GET \
+     -H "@$BEARER_TOKEN_FILE" \
+     https://app.rkvst.io/archivist/v2/assets/<asset-id>:publicurl
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+4. The following screenshot shows the public view of the Asset when the link is followed. 
+{{< img src="PublicView.png" alt="Rectangle" caption="<em>Public View</em>" class="border-0" >}}
+
+
+### Adding an Event to a Public Asset
```

Comment:
Question: AFAIK creating an event against a public asset (when logged in as the owning tenant) is the same as for a normal, permissioned asset (which is documented in "Creating an Event Against an Asset" as you pointed out in step 1.

Is duplicating this information mostly a convenience thing for readers?

Reply:
Yes, this is just a continuing example for readers to follow. It also allows us to show that the event on a public asset is automatically listed in public view. I think the more clear/explicit we can make the information, the better; especially with public assets/events being forever accessible after they're published.